interface/themes/oemr-bs3-override.css

```diff
@@ -0,0 +1,88 @@
+/*override bootstrap 3 for openemr and some new classes all prefixed with oe-*/
```

Comment:
get rid of this file (there are only 2 other themes now to support, light and tan). 

Place the following into core.css:
.form-group
.form-horizontal .control-label

Then the following would go into the individual themes:
legend
fieldset
(note that while playing around I removed the background/background-color entries in each of these elements in light style and actually didn't look that bad albeit the shading is more to my taste; this is an example why this should be left to the theme developers)

And the rest seem to all be theme specific (ie. most don't even need to be included in the other themes, but that can be on a case by case basis; for example, the oe* stuff may be needed, but I think the button building and button spacing should be left to the themes discretion (for example, I am pretty sure Robert would likely just have the straight cancel links on his light theme without the spacing); and then the responsibility of enforcing rtl for each theme can be done via demand; so this means the current themes will maintain rtl support while your new themes will require some work if a user/developer wishes to pursue this.

Reply:
Would  suggest leaving the oemr-bs3-override.css in place, at least for now, as I have now restyled and made responsive about 45 files. Many of the files needed some additional styling to achieve consistency in display. This is in keeping with your desire to cut out inline styling and minimize internal stylesheets. 
I could move .form-group .form-horizontal .control-label if you would like but I guess some others would also need to be moved. 
Choosing what and when is the tricky part ;)

Reply:
hi @zbig01 ,
To clarify, oemr-bs3-override.css, is not used in your new themes, though?
If this is the case, then as suggested above, you can remove all the other themes except light and tan. And then, might as well just copy everything in oemr-bs3-override.css(and nothing to core.css) to the top of these 2 themes for now(and make clear it's a section for new theme work) and then can remove this file. This way, you pass off complete control of these elements to the tan and light themes and their respective authors (for example, Robert for light will likely not want to use any of your cancel button stuff, but he can decide that; and Ray for tan will likely want to use your cancel stuff, but he can decide that). Doing this also will make it much more clear for folks to support RTL. And more importantly, it gets the code to a point where we can start getting it into the codebase (goal for next chunk of code into the codebase is new themes, drop old themes(discussed above which to drop), and the soap form; then next chunk will be more forms etc.).

Reply:
Also, since we are removing all themes except tan and light, then no need to have flat in your theme titles. Thoughts are:
Light will stay Light
Change Tan to ___ ( @ophthal , any thoughts on what you wish to change your theme name to; would rather have it not be a color name)
And remove flat in all your themes.